CreeDictionary/CreeDictionary/templatetags/creedictionary_extras.py

```diff
@@ -5,18 +5,52 @@
 Template tags related to the Cree Dictionary specifically.
 """
 
+from constants import DEFAULT_ORTHOGRAPHY, ORTHOGRAPHY_NAME
 from cree_sro_syllabics import sro2syllabics
 from django import template
-from django.utils.html import conditional_escape
-from django.utils.safestring import mark_safe
+from django.utils.html import format_html
 
 CIRCUMFLEX_TO_MACRON = str.maketrans("êîôâ", "ēīōā")
 
 register = template.Library()
 
 
+@register.simple_tag(name="orth", takes_context=True)
```

Comment:
I made a new tag, as opposed to updating the existing filter, because filters cannot take in `context`. Since the `orth` cookie must be accessed through `context`, then this functionality _must_ be used from a tag.